### DISCUSSION TITLE
A separate page for Race Control messages

### DISCUSSION THREAD

**Discussion by Psybertr0n** [f1multiviewer/issue-tracker#28] [Feature requests] [answered]
Can we have a separate page just for the stream of Race Control messages that is on the Live Timing page right now? I prefer to have the Data Channel video stream over the Live Timing page because I can see animations of drivers changing positions but I cannot resize the Live Timing page (even with zooming in or out) to only have the Race Control messages in view.

**Comment by f1multiviewer** [comment, marked as answer]
Fixed as of v1.0.0-beta.10!

**Reply by VictorColomb** [reply]
Currently, after popping out the RCM window, closing the live telemetry window closes the RCM one as well. Would it be possible to keep the RCM window open without the live telemetry one ?

**Reply by JustJoostNL** [reply]
@VictorColomb  https://github.com/f1multiviewer/issue-tracker/discussions/72

**Reply by beerse** [reply]
My issue with the live telemetry window is that it has no 'underscore' button at the right top, so I cannot hide it. My workaround here is to move the window off-side and behind an other 'always on top' window.
It would be nice if this window can somehow be closed and still have the RCM or the live-data at the onbord-cameras.

**Reply by f1multiviewer** [reply]
@beerse the reason it's currently unable to minimise it, is because that causes "background throttling", causing the live timing to seize up and stop working after some time. It's a workaround for now, but in the future, it might be possible to work around this :)
